contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -258,15 +258,25 @@ including configuration settings provided by the user,
 or otherwise changed after creation by other ecosystem components (e.g.,
 schedulers, auto-scalers), and is persisted in stable storage with the API
 object. If the specification is deleted, the object will be purged from the
-system. The status summarizes the current state of the object in the system, and
-is usually persisted with the object by automated processes but may be
-generated on the fly. At some cost and perhaps some temporary degradation in
-behavior, the status could be reconstructed by observation if it were lost.
+system.
+
+The status summarizes the current state of the object in the system, and is
+usually persisted with the object by automated processes but may be generated
+on the fly.  As a general guideline, fields in status should be the most recent
+observations of actual state, but they may contain information such as the
```

Comment:
```suggestion
on the fly.  As a general guideline, fields in the status should be the most recent
observations of actual state, but they may contain information such as the
```
Or maybe:
```suggestion
on the fly.  As a general guideline, fields in the status stanza should be the most recent
observations of actual state, but they may contain information such as the
```
Oh, but then in the next paragraph, you use "spec" and "status" with no preceding article. Later in the document you distinguish the words as `spec` and `status`. Quoting or distinguishing their typeface would help here.

Reply:
I feel like field names should be backticked, but we don't have a consistent convention on that.   I'll add a commit to do that and see if it is better.